ProcesoCierreIteración.md

```diff
@@ -9,7 +9,7 @@ Métricas utilizadas | Número de patrones y antipatrones presentes en la iterac
 Repositorio de Métricas | Tabla de patrones y antipatrones en el documento de [Cierre de Iteración](https://drive.google.com/open?id=1czEdn-xYM2REV3wG9aS1eavqmrkX1tWFAqdNuSrUwk8) en la wiki de cada equipo. Actualizado cada fin de iteración por el team leader.
 Entrada del Proceso | Reporte de estatus del proyecto y solución potencialmente consumible.
 Criterios de entrada | Reporte de estatus actualizado, lleno y con datos reales.
-Definir políticas y estándares | Usar plantilla [Cierre de Iteración](https://drive.google.com/open?id=1czEdn-xYM2REV3wG9aS1eavqmrkX1tWFAqdNuSrUwk8). El showcase debe de durar menos de una hora y la retrospectiva debe durar entre 30 a 60 minutos.
+Definir políticas y estándares | Usar plantilla [Cierre de Iteración](https://drive.google.com/open?id=1czEdn-xYM2REV3wG9aS1eavqmrkX1tWFAqdNuSrUwk8). El showcase debe de durar menos de una hora y la retrospectiva debe durar entre 30 a 60 minutos. El nombre del documento para la wiki de equipo debe ser como el siguiente ejemplo: "**Retrospectiva primera iteración**".
```

Comment:
El documento se llama bob the builder, y es un formato entonces debería de estar en la wiki